docs/send-data/opentelemetry-collector/data-source-and-configurations/collect-logs.md

```diff
@@ -0,0 +1,204 @@
+---
+id: collect-logs
+title: Collect Logs
+sidebar_label: Collect Logs
+description: Learn how to collecting logs
+---
+
+Sumo Logic Distribution for OpenTelemetry Collector provides various receivers for log collection. Full list of them is available in [our repository].
+This document explains with more details, the receivers which are the most commonly used for logs:
+
+* [Filelog Receiver](#filelog-receiver)
+* [Windows Log Event Receiver](#windows-log-event-receiver)
+
+## Filelog Receiver
+
+The Filelog Receiver tails and parses logs from files.
+
+The following is a basic configuration for the Filelog Receiver, which can be placed in `conf.d` directory:
+
+```yaml
+receivers:
+  filelog/custom_files:
+    include:
+    - /var/log/myservice/*.log
+    - /other/path/**/*.txt
+    - ./tmp/logs/*.log
+    include_file_name: false
+    include_file_path_resolved: true
+processors:
+  groupbyattrs/file path resolved:
+    keys:
+      - log.file.path_resolved
+service:
+  pipelines:
+    logs/custom_files:
+      receivers:
+      - filelog/custom_files
+      processors:
+      - memory_limiter
+      - groupbyattrs/file path resolved
+      - resourcedetection/system
+      - sumologic_schema
+      - batch
+      exporters:
+      - sumologic
+```
+
+The `include_file_name: false` prevents the receiver from adding `log.file.name` attribute to the logs.
+Instead, we are using `include_file_path_resolved: true`,
+which adds a `log.file.path_resolved` attribute to the logs
+that contains the whole path of the file, as opposed to just the name of the file.
+The `log.file.path_resolved` attribute should be moved to resource and we use [Group by Attributes processor][groupbyattrprocessor] for that.
+
+The remaining processors in pipeline are from `sumologic.yaml` file and should be applied for better performance of collector and use of Sumo Logic platform.
+
+:::note
+Receiver (`filelog/custom_files`) and pipeline (`logs/custom_files:`) names should be unique across all configuration files to avoid conflicts and unexpected behavior.
+:::
+
+### Keeping track of position in files
+
+By default, the Filelog receiver watches files starting at their end
+(`start_at: end` is the [default][filelogreceiver_readme]),
+so nothing will be read after the otelcol process starts until new data is added to the files.
+To change this, add `start_at: beginning` to the receiver's configuration.
+To prevent the receiver from reading the same data over and over again on each otelcol restart,
+also add the [File Storage extension][filestorageextension_docs] which is already defined in `sumologic.yaml`, that will allow Filelog receiver to persist the current
+position in watched files between otelcol restarts. Here's an example of such configuration:
```

Comment:
```suggestion
By default, the Filelog receiver watches files starting at their end (`start_at: end` is the [default][filelogreceiver_readme]), so nothing will be read after the otelcol process starts until new data is added to the files. To change this, add `start_at: beginning` to the receiver's configuration.

To prevent the receiver from reading the same data over and over again on each otelcol restart, also add the [File Storage extension][filestorageextension_docs] which is already defined in `sumologic.yaml`, that will allow Filelog receiver to persist the current position in watched files between otelcol restarts. Here's an example of such configuration:
```